packages/core/src/shared/languageServer/lspResolver.ts

```diff
@@ -0,0 +1,352 @@
+/*!
```

Comment:
can these modules live in `shared/lsp/` instead of `shared/languageServer/` ? helps discovery.